docs/introduction/prerequisites.md

```diff
@@ -0,0 +1,454 @@
+    <h3>5. Apply PATH changes immediately:</h3>
+    <pre><code>source ~/.profile</pre></code>
+    <h3>6. Check the Go installation and whether it is the supported version:</h3>
+    <pre><code>go version</pre></code>
+    <blockquote><strong>Note:</strong> For Debian/Ubuntu, you can install Go using Snap:
+    <pre><code>sudo snap install --classic go</pre></code></blockquote>
+  </details>
+
+<details>
+  <summary>Installation on macOS</summary>
+  <h3>1. Download the Go installation file:</h3>
+  <a href="https://go.dev/dl/go1.23.3.darwin-arm64.pkg">Apple macOS (ARM64), macOS 11 or later</a>
+  <a href="https://go.dev/dl/go1.23.3.darwin-amd64.pkg">Apple macOS (Intel), macOS 10.15 or later</a>
+  <h3>2. Run the downloaded file and follow the installation instructions.</h3>
+  <p>The package installs the Go distribution into `/usr/local/go`. The installer should add the `/usr/local/go/bin` directory to the `PATH` environment variable. You may need to restart any open Terminal sessions for the change to take effect.</p>
+  <h3>3. Check your Go installation and whether it is the supported version:</h3>
+  <pre><code>go version</pre></code>
+</details>
+
+<p>If you have any questions or problems installing Go, please consult the <a href="https://go.dev/doc/install">official documentation</a>.</p>
+
+</details>
+
+## Install Helm
+
+<details>
+  <summary>About Helm</summary>
+  <h3>What is Helm?</h3>
+  <p>Helm is a package manager for Kubernetes, the platform that automates deployment, scaling, and management of containerized applications.</p>
+  
+  <h3>Why is Helm necessary?</h3>
+  <p>In the <strong>External Secrets Operator</strong> project, Helm is used to simplify the installation and management of applications within Kubernetes, automating complex configuration and deployment processes.</p>
+</details>
+
+<details>
+  <summary>Installing Helm</summary>
+  <h3>Required Version</h3>
+  <p><strong>Recommended version:</strong> Helm 3 (latest version of Helm 3).</p>
+  <h3>How to Install Helm</h3>
+
+  <details>
+    <summary>Installing on Linux</summary>
+    <h3>1. Download the installation script:</h3>
+    <pre><code>curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3</code></pre>
+    <h3>2. Make the script executable:</h3>
+    <pre><code>chmod 700 get_helm.sh</code></pre>
+    <h3>3. Run the installation script:</h3>
+    <pre><code>./get_helm.sh</code></pre>
+    <h3>4. Verify the Helm installation:</h3>
+    <pre><code>helm version</code></pre>
+  </details>
+
+  <details>
+    <summary>Installing on macOS</summary>
+    <h3>1. Using Homebrew:</h3>
+    <pre><code>brew install helm</code></pre>
+    <h3>2. Verify the Helm installation:</h3>
+    <pre><code>helm version</code></pre>
+  </details>
+
+  <p>For other installation options and details, refer to the <a href="https://helm.sh/docs/intro/install/">official Helm installation guide</a>.</p>
+</details>
+
+---
+
+## Install yq
```

Comment:
Link to official doc instead.

Reply:
Same for the rest the tools. Just plain link to their respective official guides. :)